### ISSUE TITLE
In the paper, is the pseudo-label of the ImageNet the classification label? or bounding box annotation for ImageNet dataset? (In object detection)

### ISSUE THREAD

**Issue by woojung-son** [tensorflow/tpu#1019]
In the paper, the self-training pipeline is described as follows:
1) train the teacher model on the target dataset (COCO)
2) Generate the pseudo labels for ImageNet dataset
3) Initialize the student model with the result from 1)
4) Train the student model with pseudo-labeled ImageNet dataset and COCO dataset.

In 2), is the pseudo label the classification label? or bounding box annotation for ImageNet dataset?
